lte/gateway/python/magma/pipelined/app/enforcement_stats.py

```diff
@@ -581,7 +576,11 @@ def get_stats(self, cookie: int = 0, cookie_mask: int = 0):
             self.logger.error("Could not initialize datapath for stats retrieval")
             return RuleRecordTable()
         parser = self._datapath.ofproto_parser
-        message = parser.OFPFlowStatsRequest(datapath=self._datapath, cookie = cookie, cookie_mask = cookie_mask)
+        if (cookie == -1):
```

Comment:
hm what is this for? If you don't want to match on cookie you can just give 0 for cookie_mask

Reply:
According to Pravin, getting all flows would be not passing anything at all. I assumed since cookie mapped directly to shard that I needed to not pass anything to poll all shards.

Reply:
So when I asked Pravin about polling all shards he mentioned that it meant not passing in any cookie value at all. Since a shard id of 0 would ideally be directly linked to a cookie value of 0, I assumed that meant I should just pass in nothing.